Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/impl/EventLogServiceImpl.java

```diff
@@ -475,6 +475,13 @@ public boolean canUserWriteLog(String username, Integer logId) throws UserNotFou
 	return false;
     }
 
+	@Override
+	public boolean canUserReadLog(String username, Integer logId) throws UserNotFoundException {
+		User user = userSrv.findUserByLogin(username);
+		return groupLogRepo.findByLogAndUser(logId, user.getRowGuid()).stream()
+			.anyMatch(gl -> gl.getAccessRights().isReadOnly() || gl.getAccessRights().isOwnerShip());
```

Comment:
My goodness, the accessor method names on the `AccessRights` class are dangerously confusing (isReadOnly rather than hasReadPermission).  But this looks correct despite that.